src/resources/html-attributes.ts

```diff
@@ -0,0 +1,17 @@
+// this function will ensure that we propertly treat a potential string value for a boolean attribute
+// as the boolean representation
+export function normalizeBooleanAttribute(attributeName: string, value: boolean | string): boolean {
```

Comment:
This may feel nitpicky, but can we change the comment to a jsdoc style comment. A small change, but this would give us IntelliSense support in most editors.

``` javascript
/*
 * This function will ensure that we properly treat a potential string
 * value for a boolean attribute as the boolean representation
 */
```